testing/src/test/java/org/oppia/android/testing/CoroutineExecutorServiceTest.kt

```diff
@@ -1091,23 +1092,6 @@ class CoroutineExecutorServiceTest {
     }
   }
 
-  // TODO(#89): Move to a common test library.
-  private fun <T : Throwable> assertThrows(type: KClass<T>, operation: () -> Unit): T {
```

Comment:
we are removing from 3 files, but I didn't see how we are using our newly introduced file?
`import org.oppia.android.testing.assertThrows`

Reply:
The import isn't required because the file are in the same module.

Reply:
ah, missed it. Thanks